modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+
+{gitops-title} 1.10.0 is now available on {product-title} 4.12 and 4.13.
+
+[id="errata-updates-1-10.0_{context}"]
+== Errata updates
+
+[id="gitops-1-10-0-security-update-advisory_{context}"]
+=== RHSA-2023:XXXX and RHEA-2023:YYYY - {gitops-title} 1.10.0 security update advisory
+
+Issued: 2023-09-26
+
+The list of security fixes and enhancements that are included in this release is documented in the following advisories:
+
+* link:https://errata.devel.redhat.com/advisory/120274[RHSA-2023:XXXX]
+* link:https://errata.engineering.redhat.com/advisory/120119[RHEA-2023:YYYY]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="new-features-1-10-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the Argo CD CRD API version is upgraded from `v1alpha1` to `v1beta1` in order to accomodate the breaking changes resulting from the deprecation of `.spec.dex` and certain `.spec.sso` fields. To streamline the automatic migration of existing `v1alpha1` Argo CD CRs to `v1beta1`, conversion webhook support has been implemented. link:https://issues.redhat.com/browse/GITOPS-3040[GITOPS-3040]
++
+[NOTE]
+====
+By default, the conversion webhook is enabled only for OLM-installed Operators. For non-OLM installations of the Operator, enabling the webhook is optional. However, without conversion webhook support, you have to manually migrate any existing Argo CD `v1alpha1` CRs to `v1beta1`.
+====
+
+* With this update, the {gitops-title} Operator deploys three monitoring dashboards in the Administrator perspective of the web console. The three dashboards are: *GitOps Overview*, *GitOps Components*, and *GitOps gRPC*. To access these dashboards, go to *Observe* → *Monitoring*. link:https:https://issues.redhat.com/browse/GITOPS-1767[GITOPS-1767]
++
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+* With this update, the {gitops-title} Operator now gathers and exports metrics data that can be handled by Prometheus. The metrics data collected by the server include the following:
++
+** The total number of Argo CD instances managed and reconciled.
+** The number of Argo CD instances in a given state, such as pending, available, among others. 
+** The total time for the reconciliation loops to complete per instance,
+** The amount of resources used such as CPU, memory, and number of goroutines.
++
+To access these metrics, go to the *Observe* tab on the web console, and run queries against the monitoring stack. link:https://issues.redhat.com/browse/GITOPS-2645[GITOPS-2645]
++
+[NOTE]
+====
+You need to install the {gitops-title} Operator in the default `openshift-gitops-operator` namespace, in order to have these metrics automatically available.
```

Comment:
users will also need to have monitoring enabled to view the dashboards in addition to having the operator installed in that namespace. When they install from operatorhub they will have a checkbox option presented to them asking if they want to enable monitoring and it has to be checked.

Reply:
Noted